code/game/machinery/traitorous_cloner.dm

```diff
@@ -0,0 +1,355 @@
+	ADD_TRAIT(H, TRAIT_NOCRITDAMAGE, CLONING_POD_TRAIT)
+	H.Unconscious(80)
+
+	var/list/candidates = pollCandidatesForMob("Do you want to play as [clonename]'s defective clone?", null, null, null, 100, H, POLL_IGNORE_DEFECTIVECLONE)
+	if(LAZYLEN(candidates))
+		var/mob/dead/observer/C = pick(candidates)
+		H.key = C.key
+
+	if(grab_ghost_when == CLONER_FRESH_CLONE)
+		H.grab_ghost()
+		to_chat(H, "<span class='notice'><b>Consciousness slowly creeps over you as your body regenerates.</b><br><i>So this is what cloning feels like?</i></span>")
+
+	if(grab_ghost_when == CLONER_MATURE_CLONE)
+		H.ghostize(TRUE)	//Only does anything if they were still in their old body and not already a ghost
+		to_chat(H.get_ghost(TRUE), "<span class='notice'>Your body is beginning to regenerate in a cloning pod. You will become conscious when it is complete.</span>")
+
+	if(H)
+		H.faction |= factions
+
+		H.set_cloned_appearance()
+
+		H.set_suicide(FALSE)
+	attempting = FALSE
+
+
+	if(!GLOB.admin_objective_list)
+		generate_admin_objective_list()
+
+	if(!isnull(H.mind)) // If we don't have mind, don't set any objectives to it, or it will break the cloner with an infinite loop of exceptions.
+		var/datum/antagonist/antag
+		if(!antag)
+			antag = new /datum/antagonist/custom
+
+		var/datum/objective/O = new /datum/objective/custom
+		O.team = team
+		O.explanation_text = "Impersonate [clonename]."
+		//O.owner = H.mind
+
+		antag.name = "Defective Clone"
+		antag.objectives |= O
+
+		if(emmaged)
+			var/expl_text = stripped_input(usr, "Enter additional objective", "Objective:", "Free objective.")
+			if(expl_text)
+				var/datum/objective/SecO = new /datum/objective/custom
+				SecO.team = team
+				SecO.explanation_text = expl_text
+				//SecO.owner = H.mind
+				antag.objectives |= SecO
+
+		H.mind.add_antag_datum(antag, team)
+		H.mind.announce_objectives()
+
+	return CLONING_DELETE_RECORD | CLONING_SUCCESS //so that we don't spam clones with autoprocess unless we leave a body in the scanner
+
+/obj/machinery/clonepod/traitorous/go_out()
+	var/mob/living/mob_occupant = occupant
+	mob_occupant.mind.announce_objectives()
+	..()
+
+/obj/machinery/clonepod/traitorous/proc/make_emmaged()
+	emmaged = TRUE
+
+/obj/machinery/clonepod/traitorous/proc/is_emmaged()
+	return emmaged
+
+//Prototype cloning console, much more rudimental and lacks modern functions such as saving records, autocloning, or safety checks.
+/obj/machinery/computer/prototype_cloning_traitorous
+	name = "prototype cloning console traitorous"
+	desc = "Used to operate an experimental cloner."
+	icon_screen = "dna"
+	icon_keyboard = "med_key"
+	circuit = /obj/item/circuitboard/computer/prototype_cloning_traitorous
+	var/obj/machinery/dna_scannernew/scanner = null //Linked scanner. For scanning.
+	var/list/pods //Linked experimental cloning pods
+	var/temp = "Inactive"
+	var/scantemp = "Ready to Scan"
+	var/loading = FALSE // Nice loading text
+	var/is_emmaged = FALSE
```

Comment:
shouldn't it be emaged too just to be consistent with the pod?

Reply:
It was renamed specifically to not confuse them.